android/app/src/main/java/app/organicmaps/SplashActivity.java

```diff
@@ -188,11 +188,13 @@ public void processNavigation()
       result.putExtra(EXTRA_INITIAL_INTENT, initialIntent);
       if (!initialIntent.hasCategory(Intent.CATEGORY_LAUNCHER))
       {
+        /// @todo Is it ok that we don't call setFirstStartDialogSeen here?
```

Comment:
@rtsisyk

Reply:
It wasn't called before also. The todo question is not to forget or dpcument that API call is not a first call, because doesn't show onboarding dialog.